src/dbnode/storage/bootstrap/result/result_index.go

```diff
@@ -33,6 +33,11 @@ import (
 	xtime "github.com/m3db/m3/src/x/time"
 )
 
+const (
+	// Volume index defaults to -1 or unset.
+	defaultVolumeIndex = -1
```

Comment:
I think we default to 0 for data files. Should we do the same here or is it somewhat apples and oranges?

Reply:
Sorry, the naming here is confusing I changed it to the following:
```
// Volume index of -1 means unset.
volumeIndexUnset = -1
```
The default is 0 for index data too.